js/app/src/lite/ErrorDisplay.svelte

```diff
@@ -21,20 +21,19 @@
 	autoscroll={false}
 >
 	<div class="error" slot="error">
-	{#if error}
-	{#if error.message}
-		<p class="error-name">
-			{error.message}
-		</p>
-	{/if}
-	{#if error.stack}
-		<pre class="error-stack"><code>{error.stack}</code></pre>
-	{/if}
-	{/if}
+		{#if error}
```

Comment:
I ran `pnpm format:write` and it changed a bunch of stuff lol

Reply:
Anything that's not 

* Modifying an Index.svelte or package.json to export some classes
* A new or changed README
* related to SimpleAudio or SimpleVideo

Is probably just automatic lint